packages/gestalt/src/boxWhitespace.css

```diff
@@ -41,6 +41,16 @@ html[dir="rtl"] .marginEnd0 {
   margin-left: calc(var(--g-boint) * 0);
 }
 
+.paddingY0 {
+  padding-bottom: calc(var(--g-boint) * 0);
+  padding-top: calc(var(--g-boint) * 0);
+}
+
+.paddingX0 {
+  padding-left: calc(var(--g-boint) * 0);
+  padding-right: calc(var(--g-boint) * 0);
+}
+
```

Comment:
Mentioned this in Slack too, but I think we should at least add a description to the prop table about this ability (for both padding and margin), and even better would be to add an example to the documentation about responsive padding/margin

Reply:
@ayeshakmaz updated - thank you. Also added an example